btx/processing/indexer.py

```diff
@@ -176,4 +176,5 @@ def parse_input():
     if not params.report:
         indexer_obj.launch()
     else:
-        indexer_obj.report(params.update_url)
+        pass
```

Comment:
Do we need a warning here too?

Reply:
Right now, when running as workflow, it is setup so that it only gets here as the second command of the second batch script - if the new task works well we want it to pass here. I just left the code structure as is so it's easy to see what the old logic was doing. It'll either be removed entirely in the future, or be used to do the summary posting that the new task does currently.